index.html

```diff
@@ -3,9 +3,35 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="./style.css">
+    <script src="https://kit.fontawesome.com/c72a7bd198.js" crossorigin="anonymous"></script>
+    <title>Carmax - Browse used cars and new cars online</title>
 </head>
 <body>
-    
+    <header>
+        <nav>
+          <ul>
+            <li id="left" class="header">Carmax</li>
+            <li id="left"><a href="#">Shop</a></li>
+            <li id="left"><a href="#">Sell</a></li>
+            <li id="left"><a href="#">Finance</a></li>
+            <li id="left"><a href="#">More</a></li>
+            <img id="left" class="sort" src="./sort-down.svg" alt="arrow vector from flaticon">
+            <img id="right" class="user" src="./user.svg" alt="user icon from flaticon">
+            <img id="right" class="heart" src="./heart.svg" alt="heart emoji from flaticon">
+            <img id="right" class="sort" src="./sort-down.svg" alt="arrow vector from flaticon">
```

Comment:
@Afrocentriccynic The images seems broken. The paths here is incorrect because the images are inside a `img` folder. So, `./img/sort-down.svg` should work.